### PR TITLE
Add/personal blog har

### PR DESCRIPTION
**Pull request by bogiii**
## Description
The PR contains an additional portion of tests, introducing a new `personal-website.har` file.

In order to make the fetch-from-har test work and consuming data from the HAR file, the fetcher method needs to be a window fetch. Since Wix parser is a stand-alone node package, the fetch method is provided as config.

Also, the Axios package is replaced with the `fetch-node` package, mainly because fetch and node-fetch methods share the same interface.

## How has this been tested?
- unit tests
- manual testing

## Types of changes
Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: test/integration/wix/__snapshots__/fetch-from-wix-har.test.js.snap [hunk 1]
```diff
@@ -95,6 +95,569 @@ exports[`wix: empty.har 1`] = `
...
+<figure class=\\"wp-block-audio\\"><audio controls src=\\"https://music.wixstatic.com/mp3/dmPSv_DPcDH86Ium3LCRu4N4jYKKCQrhgTVsy.mp3\\"></audio></figure>
+<!-- /wp:audio -->
+
+<!-- wp:embed {\\"url\\":\\"spotify:playlist:37i9dQZF1E4lCYovezyEZT\\"} -->
+<figure class=\\"wp-block-embed\\"><div class=\\"wp-block-embed__wrapper\\">
+spotify:playlist:37i9dQZF1E4lCYovezyEZT
+</div></figure>
+<!-- /wp:embed -->
+
+<!-- wp:embed {\\"url\\":\\"\\\\u0022https://soundcloud.com/samiran-saharia/tales-from-the-other-sideoriginal-mix\\\\u0022\\"} -->
```
Comment (akirk) [inline test/integration/wix/__snapshots__/fetch-from-wix-har.test.js.snap]: Not sure about the `\\u0022` here, where does it come from? Looks like the quotes around the URL still need to be stripped.

**Review by akirk — COMMENTED** [review]
Looks good, just noticed an bug revealed in the test that probably is already in `main`.

File: test/integration/wix/__snapshots__/fetch-from-wix-har.test.js.snap (hunk 1 above)
Comment (bogiii) [reply test/integration/wix/__snapshots__/fetch-from-wix-har.test.js.snap]: Nice catch @akirk!

The data we got from the SoundCloud component has additional quote marks.

Example:
```"param_font_resolveUrl": "\"https://soundcloud.com/samiran-saharia/tales-from-the-other-sideoriginal-mix\"",```

I'll fix it right away.

File: packages/site-parsers/src/parsers/wix/components/sound-cloud.js [hunk 2]
```diff
@@ -10,9 +10,12 @@ module.exports = {
 			themeData.style.properties &&
 			themeData.style.properties.param_font_resolveUrl
 		) {
-			return createBlock( 'core/embed', {
-				url: themeData.style.properties.param_font_resolveUrl,
-			} );
+			const url = themeData.style.properties.param_font_resolveUrl.replace(
+				/^(")|(")$/g,
```
Comment (akirk) [inline packages/site-parsers/src/parsers/wix/components/sound-cloud.js]: Just curious, why the parentheses?

**Review by akirk — APPROVED** [review]
Tested, works well!

File: packages/site-parsers/src/parsers/wix/components/sound-cloud.js (hunk 2 above)
Comment (bogiii) [reply packages/site-parsers/src/parsers/wix/components/sound-cloud.js]: Hmm, I didn't see that I left the parentheses. Initially, I wanted to match and replace by group and then figure out that I don't need to complicate (I forgot to remove it). :)

Comment (bogiii) [reply packages/site-parsers/src/parsers/wix/components/sound-cloud.js]: Extra parentheses have been removed.

**Pull request merged**